Specs/Scene/ModelExperimental/ModelExperimental3DTileContentSpec.js

```diff
@@ -132,10 +128,6 @@ describe("Scene/ModelExperimental/ModelExperimental3DTileContent", function () {
   });
 
   it("renders i3dm content", function () {
```

Comment:
I know Travis hasn't passed yet, but something tells me this won't work, since the draw call is a no-op.

Reply:
Never mind? I guess it works

Reply:
I'll admit I'm not entirely sure how it works, but in `addDefaultMatchers.js` it does account for the webgl stub somehow so the test can still run.

https://github.com/CesiumGS/cesium/blob/9fcc11dfa71d6e64c49aac65202f8a5c296a66f9/Specs/addDefaultMatchers.js#L239-L257